client/dashboard/top-selling-products/index.js

```diff
@@ -94,4 +97,16 @@ class TopSellingProducts extends Component {
 	}
 }
 
-export default TopSellingProducts;
+export default compose(
+	withSelect( select => {
+		const { getReportStats, isReportStatsRequesting, isReportStatsError } = select( 'wc-admin' );
+		const endpoint = NAMESPACE + 'reports/products';
+		const query = { orderby: 'items_sold' };
```

Comment:
I think we might want to add in another TODO comment here around how the date parameters for this query will be driven by the Date Picker that will be shown on the top of the dashboard page.